backend/models/airline.js

```diff
@@ -0,0 +1,18 @@
+const mongoose = require('mongoose');
+
+const airlineSchema = new mongoose.Schema(
+  {
+    name: { type: String, required: true },
+  },
+  { timestamps: true },
+);
+
+airlineSchema.set('toJSON', {
+  transform: (document, returnedObject) => {
+    returnedObject.id = returnedObject._id.toString();
```

Comment:
as I know mongoose convert ObjectId to string authomatically. You dont need to do it

Reply:
Why dont you like '_id'?